src/Symfony/Component/Form/Resources/translations/validators.hy.xlf

```diff
@@ -4,15 +4,15 @@
         <body>
             <trans-unit id="28">
                 <source>This form should not contain extra fields.</source>
-                <target>Այս ձևը չպետք է պարունակի լրացուցիչ տողեր.</target>
+                <target>Այս ձևը չպետք է պարունակի լրացուցիչ տողեր։</target>
```

Comment:
This is funny. Google translates this to: 

"This egg should ..."

 =)

Reply:
@Nyholm Tobias, in Armenian language there is a reverse syntax than in English. Dot should be with double ":" and there is a "middle dot", like in ";" in software development:
Hope I'm helped you to understand.